.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -22,4 +22,6 @@ about: Tell us about a problem you are experiencing
 - Kubernetes version: (use `kubectl version`):
 - OS (e.g. from `/etc/os-release`):
 
-/kind bug
\ No newline at end of file
+/kind bug
+[One or more /area label. See [labels list](https://github.com/kubernetes-sigs/cluster-api/labels?q=area)]
```

Comment:
potentially, add a def. article (the) `See the [labels list]...`

Reply:
@neolit123 fixed so it will look fine also not in preview + added the article